src/app/payment-form/payment-form.component.scss

```diff
@@ -1,52 +1,120 @@
-.form-card {
-  width: 100%;
-  max-width: 70%;
-  margin: 30px auto;
-  padding: 30px 0;
-
-  // Remove card styling for smaller displays.
-  @media screen and (max-width: 620px) {
-    box-shadow: none;
-    margin-top: 15px;
-    background-color: transparent;
-    padding: 0 0 30px;
-    mat-form-field {
-      width: 100%;
+::ng-deep {
+  .mat-form-field-underline {
+    display: none;
+  }
+
+  .mat-form-field-wrapper {
+    padding: 1px 8px 8px 24px;
+  }
+
+  mat-datepicker-toggle > .mat-focus-indicator.mat-icon-button.mat-button-base {
+    width: 48px;
+    height: 48px;
+    border-radius: 50%;
+    background-color: #E0DBF9;
+  }
+
+  .mat-select-arrow-wrapper {
+    width: 48px;
+    height: 48px;
+    border-radius: 50%;
+    background-color: #E0DBF9;
+    transform: translateY(-17px);
+
+    .mat-select-arrow {
+      margin: 0 19px;
     }
   }
+
+  .mat-form-field-type-mat-input .mat-form-field-infix,
+  .mat-form-field-type-mat-select .mat-form-field-infix {
+    border: 0;
+    margin-top: 17px;
+  }
+}
+
+.form-card {
   &__content {
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
-    align-items: center;
     form {
       display: flex;
       flex-direction: column;
       width: 100%;
-      max-width: 400px;
+
+      .mat-subtitle {
+        font-size: 14px;
+        margin-right: 0;
+        margin-left: 0;
+        margin-bottom: 0;
+        color: #59537B;
+      }
+
+      .mat-subtitle {
+        margin-top: 5px;
+      }
+    }
+
+    .mat-form-field-type-mat-input, .mat-form-field-type-mat-select {
+      width: 48%;
+      height: 64px;
+      border: 1px solid #E6E8EA;
+      box-sizing: border-box;
+      border-radius: 100px;
+      margin-top: 32px;
+
+      div.mat-form-field-wrapper {
+        padding: 10px;
+      }
     }
+
+    .mat-form-field-wrapper {
+      padding: 10px;
+    }
+
     .form-group-container {
       display: flex;
       justify-content: space-between;
       flex-wrap: wrap;
     }
+
     .mat-caption {
       text-align: center;
       margin-top: 0.5em;
       font-size: 0.8em;
     }
+
     .mat-title {
       margin: 0.5em 0;
     }
-    mat-datepicker-toggle {
-      position: absolute;
-      right: 0;
-      bottom: -5px;
-    }
+
     button[type=submit] {
```

Comment:
Please set our primary purple for background for this button.

Reply:
As I understand this style color provided in the HTML `color="primary"`

Reply:
Yes, but the primary color currently isn't rightfoot's purple, it is pulled from angular material's predefined theme
![image](https://user-images.githubusercontent.com/59651445/102640836-c190eb80-4163-11eb-9ce3-209648be5b1e.png)